scalikejdbc-core/src/main/scala/scalikejdbc/DataSourceConnectionPool.scala

```diff
@@ -38,5 +54,30 @@ class AuthenticatedDataSourceConnectionPool(
       settings = ConnectionPoolSettings(driverName = settings.driverName)
     ) {
 
+  def this(dataSource: DataSource, user: String, password: String) {
```

Comment:
What I care about break binary compatibility. But if it does not need care that, I will change closer: `DataSourceCloser = DefaultDataSourceCloser` as Default Parameters.